src/Console/Command/AttachmentMigrateCommand.php

```diff
@@ -169,9 +169,25 @@ private function moveFile($file): void
         $file_path = AttachmentManager::generatePath($iaf_id, $filename, $issue_id);
         $new_path = str_replace("{$this->sm->getDefaultAdapter()}://", "{$this->target_adapter}://", $file_path);
 
-        // throws League\Flysystem\Exception
-        // we let it abort whole process
-        $this->sm->moveFile($old_path, $new_path);
+        // Check if file exists. If it does
+        // run SHA-256 hash on old_path and new_path data
+        // Delete old_path if new_path verifies
+        // Recopy old_path to new_path if hashes match
+        //   after deleting new_path (invalid file)
+        if(!$this->sm->fileExists($new_path)){
```

Comment:
```suggestion
        if (!$this->sm->fileExists($new_path)) {
```

follow psr code style. there should be spaces